charts/core/templates/CRD/servicemonitor.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.global.servicemonitor.enabled -}}
+{{- if .Values.global.monitoring.servicemonitor.enabled -}}
```

Comment:
Today at least lnm-engine and Traefik are are using ServiceMonitor. Can you please make sure that there is no regression in those due to this change?